### PR TITLE
Add view and router to career page

### PR DESCRIPTION
**Pull request by veredrec**
Relates #29

### CODE REVIEW

**Review by matthewdking — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: src/controllers/career.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// const career = require('./../../database/career_model.js');
```
Comment (Karyum) [inline src/controllers/career.js]: why is this whole thing commneted out ?

**Review by Heathercoraje — APPROVED** [review]

**Comment by Karyum** [comment]
@veredrec conflict!!

**Review by Karyum — APPROVED** [review]
# :heart:

**Comment by Heathercoraje** [comment]
fin-fucking-ally

**Comment by Karyum** [comment]
@Heathercoraje i saw that

**Pull request merged**